src/towncrier/check.py

```diff
@@ -57,15 +57,28 @@ def _get_default_compare_branch(branches: Container[str]) -> str | None:
     metavar="FILE_PATH",
     help=config_option_help,
 )
-def _main(compare_with: str | None, directory: str | None, config: str | None) -> None:
+@click.option(
+    "--staged",
+    "staged",
+    is_flag=True,
+    default=False,
+    metavar="STAGED",
```

Comment:
Does `metavar` do anything when it's a flag?

Reply:
good point. removed in: de88e69f9eb2aecfefcfe259250d093a2bf2dcc9